pkg/sql/plan/build_util.go

```diff
@@ -137,6 +136,10 @@ func buildDefaultExpr(col *tree.ColumnTableDef, typ *plan.Type) (*plan.Default,
 		}, nil
 	}
 
+	if typ.GetId() == int32(types.T_blob) {
+		return nil, errors.New(errno.InvalidColumnDefinition, fmt.Sprintf("Type text don't support default value"))
```

Comment:
Why?

Reply:
because in the db, blob type now is text type, text type don't support default value.

Reply:
why is that?   If varchar can have default values, why not text?